spoc/pixels.py

```diff
@@ -55,61 +70,189 @@ def __init__(
             self._path = Path(pixel_source)
             self._data = None
 
+    @staticmethod
+    def from_uri(uri, mode="path"):
```

Comment:
Same from `Contacts`: we add IO functionality here instead of leaving it to the `FileManager`.

Reply:
Moved to FileManager